data/aliases.js

```diff
@@ -2,29 +2,29 @@
 
 exports.BattleAliases = {
 	// formats
-	"randbats": "[Gen 7] Random Battle",
-	"uber": "[Gen 7] Ubers",
+	"2v2": "[Gen 7] 2v2 Doubles",
+	"aaa": "[Gen 7] Almost Any Ability",
 	"ag": "[Gen 7] Anything Goes",
-	"randdubs": "[Gen 7] Random Doubles Battle",
-	"doubles": "[Gen 7] Doubles OU",
-	"dou": "[Gen 7] Doubles OU",
-	"vgc17": "[Gen 7] VGC 2017",
-	"vgc": "[Gen 7] VGC 2017",
-	"bss": "[Gen 7] Battle Spot Singles",
+	"bh": "[Gen 7] Balanced Hackmons",
+	"bf": "[Gen 6] Battle Factory",
 	"bsd": "[Gen 7] Battle Spot Doubles",
-	"bsdoubles": "[Gen 7] Battle Spot Doubles",
+	"bss": "[Gen 7] Battle Spot Singles",
 	"bstriples": "[Gen 6] Battle Spot Triples",
+	"bsdoubles": "[Gen 7] Battle Spot Doubles",
```

Comment:
this should still be under "bsd" because the third letter in bsdoubles still comes before the third letter in bss alphabetically